### PR TITLE
fix: swap UAT CR

### PR DESCRIPTION
**Pull request by SorinC6**
task:
- https://emurgo.atlassian.net/browse/YOMO-973
- https://emurgo.atlassian.net/browse/YOMO-971
- https://emurgo.atlassian.net/browse/YOMO-970 - for the cards box shadow not sure how to create the exact shadow as in the design-> from what I see there will always be a small difference between IOS/ANDROID on this matter 
- https://emurgo.atlassian.net/browse/YOMO-969
- https://emurgo.atlassian.net/browse/YOMO-976
- https://emurgo.atlassian.net/browse/YOMO-905 - change precision to 10
- https://emurgo.atlassian.net/browse/YOMO-977 - force/add gray color when input is 0

### CODE REVIEW

File: apps/wallet-mobile/src/features/Swap/common/helpers.ts [hunk 1]
```diff
@@ -99,3 +99,11 @@ export const sortTokensByName = (a: Balance.TokenInfo, b: Balance.TokenInfo, wal
 
   return nameA.localeCompare(nameB, undefined, {sensitivity: 'base'})
 }
+
+export const switchDayAndMonth = (dateString: string): string => {
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/common/helpers.ts]: hey @SorinC6 is there a ticket for this one? cuz this should be based on the `Intl` I don't think we should update it.

Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/common/helpers.ts]: please ping Product.

Comment (SorinC6) [reply apps/wallet-mobile/src/features/Swap/common/helpers.ts]: @stackchain  the fix is for this task: https://emurgo.atlassian.net/browse/YOMO-976 . Did not find a way to made it as it the design only if switching them

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/common/helpers.ts]: If we can tell the formatter to use 2 digits for years, that may be done. But changing the order of the months and days or whatever... that's locale based, not design based.

File: apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ export const AmountCard = ({
               onChangeText={onChange}
               allowFontScaling
               selectionColor={isFocused ? '#242838' : COLORS.TRANSPARENT_BLACK}
-              style={styles.amountInput}
+              style={[styles.amountInput, value === '0' && styles.grayText]}
```
Comment (jorbuedo) [inline apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx:73]: I don't think this one's what they want.

Comment (SorinC6) [reply apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx:73]: have gone back and forward regarding it - they want the grey color even when the input is 0 .. at least this is what I understand... can discuss more related to it here https://emurgo.atlassian.net/browse/YOMO-977 @jorbuedo

File: apps/wallet-mobile/src/features/Swap/common/helpers.ts (hunk 1 above)
Comment (SorinC6) [reply apps/wallet-mobile/src/features/Swap/common/helpers.ts]: @jorbuedo not sure how to achieve it without switching the order 🤔

Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Swap/common/helpers.ts]: We shouldn't be deciding any order. There's date formatters that take into account the locale. I'm pretty sure javascript comes with one by default. But we also have one installed 
`@formatjs/intl-datetimeformat` So whatever that thing says goes.

Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/common/helpers.ts]: can you check it @SorinC6 ?

**Review by stackchain — APPROVED** [review]

**Pull request merged**